{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2017-12-20T02:19:56.182700Z",
     "start_time": "2017-12-20T02:18:35.238700Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "WARNING (theano.configdefaults): g++ not available, if using conda: `conda install m2w64-toolchain`\n",
      "D:\\App\\anaconda3\\lib\\site-packages\\theano\\configdefaults.py:560: UserWarning: DeprecationWarning: there is no c++ compiler.This is deprecated and with Theano 0.11 a c++ compiler will be mandatory\n",
      "  warnings.warn(\"DeprecationWarning: there is no c++ compiler.\"\n",
      "WARNING (theano.configdefaults): g++ not detected ! Theano will be unable to execute optimized C-implementations (for both CPU and GPU) and will default to Python implementations. Performance will be severely degraded. To remove this warning, set Theano flags cxx to an empty string.\n",
      "WARNING (theano.tensor.blas): Using NumPy C-API based implementation for BLAS functions.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Theano version 1.0.1\n",
      "theano is installed in D:\\App\\anaconda3\\lib\\site-packages\\theano\n",
      "NumPy version 1.13.3\n",
      "NumPy relaxed strides checking option: True\n",
      "NumPy is installed in D:\\App\\anaconda3\\lib\\site-packages\\numpy\n",
      "Python version 3.6.3 |Anaconda custom (32-bit)| (default, Oct 15 2017, 07:29:16) [MSC v.1900 32 bit (Intel)]\n",
      "nose version 1.3.7\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "EE...........E..S......D:\\App\\anaconda3\\lib\\site-packages\\theano\\compile\\nanguardmode.py:150: RuntimeWarning: All-NaN slice encountered\n",
      "  return np.isinf(np.nanmax(arr)) or np.isinf(np.nanmin(arr))\n",
      "D:\\App\\anaconda3\\lib\\site-packages\\theano\\compile\\nanguardmode.py:150: RuntimeWarning: All-NaN axis encountered\n",
      "  return np.isinf(np.nanmax(arr)) or np.isinf(np.nanmin(arr))\n",
      ".E...................................................SSSSSSSSS...SSS..........S.......E..S............................................S.............................................E........SS.S........S.ED:\\App\\anaconda3\\lib\\site-packages\\theano\\gof\\vm.py:889: UserWarning: LoopGC does not support partial evaluation, using Stack VM.\n",
      "  'LoopGC does not support partial evaluation, '\n",
      "SS...SEEEESEEEEEEEESEEEES.D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\type.py:861: UserWarning: config.experimental.unpickle_gpu_on_cpu is set to True. Unpickling GpuArray as numpy.ndarray\n",
      "  \"config.experimental.unpickle_gpu_on_cpu is set to True. \"\n",
      ".EEEEEES.....SEE...D:\\App\\anaconda3\\lib\\site-packages\\theano\\sandbox\\rng_mrg.py:1031: UserWarning: MRG_RandomStreams.multinomial_wo_replacement() is deprecated and will be removed in the next release of Theano. Please use MRG_RandomStreams.choice() instead.\n",
      "  warnings.warn('MRG_RandomStreams.multinomial_wo_replacement() is '\n",
      "....EES....E..S.E.......EEEE.EEEEEEEEEEEEEEE.EEEE.....EEEEE...E....E.....S.....EEEEEEEEE......EEE......EEEE..........SEE..SSSS.......S.....E...\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\compile\\tests\\test_builders.py\", line 17, in <module>\n",
      "    from theano.tests import unittest_tools\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\compile\\tests\\test_debugmode.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\compile\\tests\\test_function_module.py\", line 16, in <module>\n",
      "    from theano.tests.unittest_tools import SkipTest\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\compile\\tests\\test_ops.py\", line 8, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gof\\tests\\test_destroyhandler.py\", line 14, in <module>\n",
      "    from theano.tests.unittest_tools import assertFailure_fast\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gof\\tests\\test_params_type.py\", line 11, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: theano.gof.tests.test_vm.test_partial_function\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\case.py\", line 197, in runTest\n",
      "    self.test(*self.arg)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gof\\tests\\test_vm.py\", line 197, in test_partial_function\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_abstractconv.py\", line 8, in <module>\n",
      "    from theano.tensor.nnet.tests import test_abstract_conv\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_abstract_conv.py\", line 11, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_basic_ops.py\", line 14, in <module>\n",
      "    from theano.tensor.tests import test_basic\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_basic.py\", line 58, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_blas.py\", line 10, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_blocksparse.py\", line 6, in <module>\n",
      "    import theano.tests.unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_ctc.py\", line 8, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_dnn.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_elemwise.py\", line 10, in <module>\n",
      "    from theano.tests.unittest_tools import SkipTest, assert_allclose\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_extra_ops.py\", line 10, in <module>\n",
      "    import theano.tensor.tests.test_extra_ops\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_extra_ops.py\", line 6, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_fft.py\", line 7, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_gemmcorr.py\", line 7, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_gemmcorr3d.py\", line 7, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_linalg.py\", line 18, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_multinomial.py\", line 15, in <module>\n",
      "    import theano.tests.unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_neighbours.py\", line 2, in <module>\n",
      "    from theano.tensor.nnet.tests import test_neighbours\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_neighbours.py\", line 10, in <module>\n",
      "    from theano.tests import unittest_tools\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_nnet.py\", line 8, in <module>\n",
      "    import theano.tests.unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_opt.py\", line 9, in <module>\n",
      "    from theano.tests import unittest_tools as utt, test_ifelse\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_pool.py\", line 13, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_reduction.py\", line 7, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_rng_mrg.py\", line 10, in <module>\n",
      "    from theano.sandbox.tests.test_rng_mrg import java_samples, rng_mrg_overflow\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sandbox\\tests\\test_rng_mrg.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_scan.py\", line 8, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_sort.py\", line 4, in <module>\n",
      "    import theano.tensor.tests.test_sort\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_sort.py\", line 5, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\gpuarray\\tests\\test_subtensor.py\", line 8, in <module>\n",
      "    from theano.tensor.tests import test_subtensor, test_basic\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_subtensor.py\", line 34, in <module>\n",
      "    from theano.tensor.tests.test_basic import inplace_func, rand, randint_ranged\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_basic.py\", line 58, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sandbox\\linalg\\tests\\test_linalg.py\", line 8, in <module>\n",
      "    from theano.tests.test_rop import break_op\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_rop.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sandbox\\tests\\test_multinomial.py\", line 12, in <module>\n",
      "    import theano.tests.unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sandbox\\tests\\test_rng_mrg.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\scalar\\tests\\test_basic.py\", line 19, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\scan_module\\tests\\test_scan.py\", line 24, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\scan_module\\tests\\test_scan_opt.py\", line 9, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sparse\\sandbox\\test_sp.py\", line 21, in <module>\n",
      "    from theano.sparse.tests.test_basic import random_lil\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sparse\\tests\\test_basic.py\", line 22, in <module>\n",
      "    from theano.tests.unittest_tools import attr\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sparse\\tests\\test_basic.py\", line 22, in <module>\n",
      "    from theano.tests.unittest_tools import attr\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sparse\\tests\\test_opt.py\", line 13, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sparse\\tests\\test_sp2.py\", line 22, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sparse\\tests\\test_utils.py\", line 8, in <module>\n",
      "    from theano.sparse.tests.test_basic import as_sparse_format\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\sparse\\tests\\test_basic.py\", line 22, in <module>\n",
      "    from theano.tests.unittest_tools import attr\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_abstract_conv.py\", line 11, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_blocksparse.py\", line 11, in <module>\n",
      "    import theano.tests.unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_bn.py\", line 4, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_conv.py\", line 8, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_conv3d2d.py\", line 5, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_corr.py\", line 11, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_corr3d.py\", line 6, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_ctc.py\", line 9, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_neighbours.py\", line 10, in <module>\n",
      "    from theano.tests import unittest_tools\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_nnet.py\", line 14, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_opt.py\", line 4, in <module>\n",
      "    from theano.tests.unittest_tools import assertFailure_fast\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\nnet\\tests\\test_sigm.py\", line 12, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\signal\\tests\\test_conv.py\", line 9, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\signal\\tests\\test_pool.py\", line 4, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_basic.py\", line 58, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_blas.py\", line 23, in <module>\n",
      "    from theano.tests import unittest_tools\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_blas_c.py\", line 20, in <module>\n",
      "    from theano.tests import unittest_tools\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_blas_scipy.py\", line 8, in <module>\n",
      "    from .test_blas import TestCase, gemm_no_inplace, TestBlasStrides\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_blas.py\", line 23, in <module>\n",
      "    from theano.tests import unittest_tools\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_complex.py\", line 7, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_elemwise.py\", line 21, in <module>\n",
      "    from theano.tests import unittest_tools\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_extra_ops.py\", line 6, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_fft.py\", line 7, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_fourier.py\", line 7, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_inc_subtensor.py\", line 4, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_keepdims.py\", line 9, in <module>\n",
      "    from theano.tests.unittest_tools import attr\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_nlinalg.py\", line 15, in <module>\n",
      "    from theano.tests.test_rop import break_op\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_rop.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_opt.py\", line 64, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_opt_uncanonicalize.py\", line 20, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_raw_random.py\", line 5, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_shared_randomstreams.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_sharedvar.py\", line 10, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_slinalg.py\", line 15, in <module>\n",
      "    from theano.tests.test_rop import break_op\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_rop.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_sort.py\", line 5, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_subtensor.py\", line 34, in <module>\n",
      "    from theano.tensor.tests.test_basic import inplace_func, rand, randint_ranged\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_basic.py\", line 58, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_var.py\", line 7, in <module>\n",
      "    import theano.tests.unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_xlogx.py\", line 8, in <module>\n",
      "    from . import test_basic as TT\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tensor\\tests\\test_basic.py\", line 58, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\typed_list\\tests\\test_basic.py\", line 16, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\typed_list\\tests\\test_type.py\", line 10, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\diverse_tests.py\", line 8, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_2nd_order_grads.py\", line 6, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_breakpoint.py\", line 5, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_gradient.py\", line 14, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_ifelse.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "======================================================================\n",
      "ERROR: Failure: ModuleNotFoundError (No module named 'parameterized')\n",
      "----------------------------------------------------------------------\n",
      "Traceback (most recent call last):\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\failure.py\", line 39, in runTest\n",
      "    raise self.exc_val.with_traceback(self.tb)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\loader.py\", line 417, in loadTestsFromName\n",
      "    addr.filename, addr.module)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 47, in importFromPath\n",
      "    return self.importFromDir(dir_path, fqname)\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\nose\\importer.py\", line 94, in importFromDir\n",
      "    mod = load_module(part_fqname, fh, filename, desc)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 235, in load_module\n",
      "    return load_source(name, filename, file)\n",
      "  File \"D:\\App\\anaconda3\\lib\\imp.py\", line 172, in load_source\n",
      "    module = _load(spec)\n",
      "  File \"<frozen importlib._bootstrap>\", line 684, in _load\n",
      "  File \"<frozen importlib._bootstrap>\", line 665, in _load_unlocked\n",
      "  File \"<frozen importlib._bootstrap_external>\", line 678, in exec_module\n",
      "  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\test_rop.py\", line 15, in <module>\n",
      "    from theano.tests import unittest_tools as utt\n",
      "  File \"D:\\App\\anaconda3\\lib\\site-packages\\theano\\tests\\unittest_tools.py\", line 7, in <module>\n",
      "    from parameterized import parameterized\n",
      "ModuleNotFoundError: No module named 'parameterized'\n",
      "\n",
      "----------------------------------------------------------------------\n",
      "Ran 415 tests in 55.239s\n",
      "\n",
      "FAILED (SKIP=37, errors=84)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<nose.result.TextTestResult run=415 errors=84 failures=0>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import theano\n",
    "\n",
    "theano.test()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2017-12-20T02:44:11.705700Z",
     "start_time": "2017-12-20T02:44:11.696700Z"
    },
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import keras"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2017-12-20T02:29:18.776700Z",
     "start_time": "2017-12-20T02:29:13.911700Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "D:\\App\\anaconda3\\lib\\site-packages\\sklearn\\cross_validation.py:41: DeprecationWarning: This module was deprecated in version 0.18 in favor of the model_selection module into which all the refactored classes and functions are moved. Also note that the interface of the new CV iterators are different from that of this module. This module will be removed in 0.20.\n",
      "  \"This module will be removed in 0.20.\", DeprecationWarning)\n"
     ]
    }
   ],
   "source": [
    "import numpy\n",
    "import pandas\n",
    "from keras.models import Sequential\n",
    "from keras.layers import Dense\n",
    "from keras.wrappers.scikit_learn import KerasClassifier\n",
    "from keras.utils import np_utils\n",
    "from sklearn.cross_validation import cross_val_score\n",
    "from sklearn.cross_validation import KFold\n",
    "from sklearn.preprocessing import LabelEncoder\n",
    "from sklearn.pipeline import Pipeline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2017-12-20T03:08:12.078700Z",
     "start_time": "2017-12-20T03:07:43.384700Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Using Theano backend.\n",
      "WARNING (theano.configdefaults): g++ not available, if using conda: `conda install m2w64-toolchain`\n",
      "D:\\App\\anaconda3\\lib\\site-packages\\theano\\configdefaults.py:560: UserWarning: DeprecationWarning: there is no c++ compiler.This is deprecated and with Theano 0.11 a c++ compiler will be mandatory\n",
      "  warnings.warn(\"DeprecationWarning: there is no c++ compiler.\"\n",
      "WARNING (theano.configdefaults): g++ not detected ! Theano will be unable to execute optimized C-implementations (for both CPU and GPU) and will default to Python implementations. Performance will be severely degraded. To remove this warning, set Theano flags cxx to an empty string.\n",
      "WARNING (theano.tensor.blas): Using NumPy C-API based implementation for BLAS functions.\n",
      "D:\\App\\anaconda3\\lib\\site-packages\\ipykernel_launcher.py:15: UserWarning: Update your `Dense` call to the Keras 2 API: `Dense(12, input_dim=8, activation=\"relu\", kernel_initializer=\"uniform\")`\n",
      "  from ipykernel import kernelapp as app\n",
      "D:\\App\\anaconda3\\lib\\site-packages\\ipykernel_launcher.py:16: UserWarning: Update your `Dense` call to the Keras 2 API: `Dense(8, activation=\"relu\", kernel_initializer=\"uniform\")`\n",
      "  app.launch_new_instance()\n",
      "D:\\App\\anaconda3\\lib\\site-packages\\ipykernel_launcher.py:17: UserWarning: Update your `Dense` call to the Keras 2 API: `Dense(1, activation=\"sigmoid\", kernel_initializer=\"uniform\")`\n",
      "D:\\App\\anaconda3\\lib\\site-packages\\keras\\models.py:939: UserWarning: The `nb_epoch` argument in `fit` has been renamed `epochs`.\n",
      "  warnings.warn('The `nb_epoch` argument in `fit` '\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Epoch 1/5\n",
      "768/768 [==============================] - 5s 6ms/step - loss: 0.6787 - acc: 0.6510\n",
      "Epoch 2/5\n",
      "768/768 [==============================] - 5s 6ms/step - loss: 0.6617 - acc: 0.6510\n",
      "Epoch 3/5\n",
      "768/768 [==============================] - 5s 6ms/step - loss: 0.6538 - acc: 0.6510\n",
      "Epoch 4/5\n",
      "768/768 [==============================] - 5s 6ms/step - loss: 0.6490 - acc: 0.6510\n",
      "Epoch 5/5\n",
      "768/768 [==============================] - 5s 6ms/step - loss: 0.6394 - acc: 0.6589\n",
      "768/768 [==============================] - 1s 1ms/step\n",
      "acc: 66.80%\n"
     ]
    }
   ],
   "source": [
    "# Create first network with Keras\n",
    "from keras.models import Sequential\n",
    "from keras.layers import Dense\n",
    "import numpy\n",
    "# fix random seed for reproducibility\n",
    "seed = 7\n",
    "numpy.random.seed(seed)\n",
    "# load pima indians dataset\n",
    "dataset = numpy.loadtxt(\"pima-indians-diabetes.csv\", delimiter=\",\")\n",
    "# split into input (X) and output (Y) variables\n",
    "X = dataset[:,0:8]\n",
    "Y = dataset[:,8]\n",
    "# create model\n",
    "model = Sequential()\n",
    "model.add(Dense(12, input_dim=8, init='uniform', activation='relu')) \n",
    "model.add(Dense(8, init='uniform', activation='relu'))\n",
    "model.add(Dense(1, init='uniform', activation='sigmoid'))\n",
    "# Compile model\n",
    "model.compile(loss='binary_crossentropy', optimizer='adam', metrics=['accuracy']) # Fit the model\n",
    "model.fit(X, Y, nb_epoch=2, batch_size=10)\n",
    "# evaluate the model\n",
    "scores = model.evaluate(X, Y)\n",
    "print(\"%s: %.2f%%\" % (model.metrics_names[1], scores[1]*100))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "ExecuteTime": {
     "end_time": "2017-12-20T03:08:22.124700Z",
     "start_time": "2017-12-20T03:08:22.104700Z"
    }
   },
   "outputs": [],
   "source": [
    "import graphviz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  },
  "toc": {
   "nav_menu": {},
   "number_sections": true,
   "sideBar": true,
   "skip_h1_title": false,
   "toc_cell": false,
   "toc_position": {},
   "toc_section_display": "block",
   "toc_window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
